website/docs/api/large-language-models.mdx

```diff
@@ -19,26 +19,20 @@ prototyping** and **prompting**, and turning unstructured responses into
 An LLM component is implemented through the `LLMWrapper` class. It is accessible
 through a generic `llm`
 [component factory](https://spacy.io/usage/processing-pipelines#custom-components-factories)
-as well as through task-specific component factories:
-
-- `llm_ner`
-- `llm_spancat`
-- `llm_rel`
-- `llm_textcat`
-- `llm_sentiment`
-- `llm_summarization`
+as well as through task-specific component factories: `llm_ner`, `llm_spancat`,
+`llm_rel`, `llm_textcat`, `llm_sentiment` and `llm_summarization`.
```

Comment:
Add entity_linker factory

Reply:
Done in https://github.com/explosion/spaCy/pull/12988/commits/c59f10cf1df55f3f3d20cdf90d7f60ef335e8ea2.